Moda.Web/src/moda.web.reactclient/src/app/components/common/organizations/create-team-form.tsx

```diff
@@ -0,0 +1,165 @@
+'use client'
```

Comment:
What do you think about moving this under an organizations/components folder rather than components/common/organizations?